devops/dependencies.json

```diff
@@ -0,0 +1,71 @@
+{
+  "linux": {
+    "compute_runtime": {
+      "github_tag": "21.46.21636",
+      "version": "21.46.21636",
```

Comment:
It seems version is not used. Can we drop it?

Reply:
This is for compatibility with [dependency.conf](https://github.com/intel/llvm/blob/6cb359522759ae01e941a94611e97582ca00ca6c/buildbot/dependency.conf#L8), which uses driver version instead of git tag (and versions and tags don't always align).